src/extension.ts

```diff
@@ -176,6 +176,13 @@ function getCurrentWorkingDirectory(): string {
 
     const isFileOrUntitledDocument = uri && (uri.scheme === 'file' || uri.scheme === 'untitled');
     if (isFileOrUntitledDocument) {
+        const useDocumentDirAsWorkDir = vscode.workspace.getConfiguration('filterText').useDocumentDirAsCwd;

```

Comment:
@mwick83, `vscode.workspace.getConfiguration('filterText').useDocumentDirAsCwd` should be `vscode.workspace.getConfiguration('filterText').useDocumentDirAsWorkDir`?

Reply:
Yes, you are right. I did a last minute change of the name and forgot the essential part ;-)